src/endpoints/v1.es6.js

```diff
@@ -144,9 +155,20 @@ class APIv1Endpoint {
     });
   }
 
+  log (state, method, uri, options, status, err, duration) {
+    if (this.config.debugLevel === 'info') {
```

Comment:
I guess we don't have integer equivalents for `>=` comparisons of debug level? Do we have a "debug" level for `debugLevel`? If so, shouldn't this also still run?